C-Plus-Plus/math/neon_number.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+	A neon number is a number where :- 
+	the sum of digits of square of the number is equal to the number. 
+	The task is to check and print neon numbers in a range given by the user.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+//Function that prints the square of a number
+int square(int z)
+{
+    return z * z;
+}
+//Function that prints the sum of digits of a number
+int sum_of_digits(int y)
+{
+    int s = 0,v;
+    while (y != 0)
+    {
+	  v=y%10;
+        s = s + v;
+        y = y / 10;
+    }
+    return s;
+}
+bool Neon(int n)
+{
+    int sq = square(n);
+    int result = sum_of_digits(sq);
+    //If sum of digits become equal to the number
+    if (result == n)
+        return true;
+    else
+        return false;
+}
+
+int main()
+{
+    cout << "Enter the range:";
+    int a, b;
+    cin >> a >> b;
+
+    // Printing Neon Numbers according to the range
```

Comment:
Print the statement what are you printing, for better user interface.
ex: The Neon numbers in the between _ and _ are :

Reply:
Resolve this @ankan1811